extension/js/common/composer.ts

```diff
@@ -545,18 +545,27 @@ export class Composer {
     });
   }
 
-  private encryptMsgAsOfDateIfSomeAreExpired = async (armoredPubkeys: string[]): Promise<Date | undefined> => {
+  private encryptMsgAsOfDateIfSomeAreExpired = async (acctEmail: string, armoredPubkeys: PubkeyResult[]): Promise<Date | undefined> => {
```

Comment:
I think I have edited this method since, please pull from master

Reply:
Yes, I've already merged it